geonode_mapstore_client/apps.py

```diff
@@ -28,5 +29,6 @@ class AppConfig(BaseAppConfig):
     label = "geonode_mapstore_client"
 
     def ready(self):
-        run_setup_hooks()
+        if not apps.ready:

```

Comment:
It looks to me that by adding this check the `run_setup_hooks()` will never be executed

Reply:
No Alessio. It still gets executed. 

1. Adds the urls. 
2. Also tested with breakpoints at different places, `api/urls.py` inclusive.

Reply:
Ok, I trust you, did not tested on my side.

Reply:
My pleasure. And I had to.